plugin/storage/es/esRollover.py

```diff
@@ -202,8 +192,12 @@ def empty_list(ilo, error_msg):
         sys.exit(0)
 
 
-def get_request_session(username, password, tls, ca, cert, key):
+def get_request_session(username, password, tls, ca, cert, key, skipHostVerify):
     session = requests.Session()
+    if skipHostVerify:
+        session.verify = False
+    if ca is not None:
```

Comment:
nit: should this be `elif`?

Reply:
good point I think they are mutually exclusive.

Reply:
I have changed it a bit. if CA is provided it will use that in the precedence of skip verify. I want to avoid accidentally using insecure connections.